CONTRIBUTING.md

```diff
@@ -51,18 +51,39 @@ by first clicking the fork button on top of the repository and cloning your fork
 
 ### Testing components locally
 
-If you're hacking on a component and want to test it in a local project, use `npm link`. For example:
+#### Testing in the documentation site
+
+If you're hacking on a component and want to test it in the documentation site follow the following steps.
+
+In this repo:
+- check out a working branch
+- run `npm run cf-link`
+
+In a site specific clone of this repo:
+
+- make sure you're on the gh-pages branch
+- run `npm run cf-link`
+- run `npm run build`
+- run `npm start`
+- open a browser at http://localhost:3000/ and navigate to the component you've changed
+```
```

Comment:
This puts the `#### Testing in an outside project` header below into code comment formatting, see the rendered file.